python/servo/devenv_commands.py

```diff
@@ -33,11 +33,32 @@ def cargo(self, params):
         if not params:
             params = []
 
+        self.ensure_bootstrapped()
```

Comment:
I don’t understand why you say this is not needed. I’ve sometimes used `git pull` then `./mach cargo check`, and the latter failed because I pulled a rust upgrade and the new Rust version was not downloaded, so the build was using the system-wide Rust install.